src/main/kotlin/xyz/crossward/entities/Entry.kt

```diff
@@ -12,6 +13,16 @@ import javax.persistence.*
 )
 data class Entry(
     @Id @Column(name = "user_id") val userId: String,
-    @Column val date: LocalDate? = null,
+    @Column val date: String? = null,
```

Comment:
Was the date not parsing correctly? If not then this change looks good to me.

Reply:
Correct